queries.sql

```diff
@@ -26,4 +26,63 @@ FROM animals
 WHERE name <> 'Gabumon';
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+/* Milestone 02*/
+-- TRANSACTION 01
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT *
+FROM animals;
+ROLLBACK;
+-- TRANSACTION 02
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] Add a select statement above line `46` so as to verify the changes were made before `Commit`